src/sagemaker/workflow/step_collections.py

```diff
@@ -318,15 +318,15 @@ def __init__(
         """
         steps = []
         if "entry_point" in kwargs:
-            entry_point = kwargs["entry_point"]
-            source_dir = kwargs.get("source_dir")
-            dependencies = kwargs.get("dependencies")
+            entry_point = kwargs.get("entry_point")
```

Comment:
do we have any default value here if "entry_point" is not present here

Reply:
Thanks for reviewing! 
As in previous line (line 320), we've already checked whether the `entry_point` is in `kwargs`, then if it goes into this `if` branch, we can be sure that the "entry_point" should exist.
```
if "entry_point" in kwargs:
  entry_point = kwargs.get("entry_point")
  ...
```

Reply:
Added default value in the new commit